darts/utils/timeseries_generation.py

```diff
@@ -385,7 +385,8 @@ def datetime_attribute_timeseries(time_index: Union[pd.DatetimeIndex, TimeSeries
             attribute+"_cos": np.cos(freq * values)
         })
     else:
-        values_df = pd.DataFrame(values)
+        values_df = pd.DataFrame({attribute: values})
```

Comment:
Good one, thanks! I suspect however we may have the same issue for the `one_hot` case (line 368), could you maybe fix this one as well while you're at it? Thanks!

Reply:
@hrzn Seems that `one_hot` case does not have this issure because the column name is set in line 392-393, which is introduced in PR #246